src/mrack/transformers/beaker.py

```diff
@@ -66,6 +66,26 @@ def _get_distro_and_variant(self, host):
 
         return (required_distro, variant)
 
+    def _get_ks_meta(self, host):
+        """
+        Get `ks_meta` value from host or provisioning config or default if not defined.
+
+        The priority is following:
+            - host
+            - provisioning-config.yaml
+            - default from provisioning config
+            - empty if not defined in provisioning config
+
+        """
+        res = self._find_value(
+            host.get(CONFIG_KEY, {}),
+            "ks_meta",
+            "kickstart_metadata",
+            host["os"],
+            default=self.config.get("kickstart_metadata", {}).get("default", ""),
```

Comment:
Defining the default this way should not be needed. This behavior is build-in  by methods called indirectly by find_value_in_config_hierarchy . Here the `default` should mean, use it if nothing else work.

Reply:
removed, I forgot.